cartography/intel/gsuite/api.py

```diff
@@ -128,7 +141,7 @@ def get_all_users(admin: Resource) -> List[Dict]:
 def load_gsuite_groups(neo4j_session: neo4j.Session, groups: List[Dict], gsuite_update_tag: int) -> None:
     ingestion_qry = """
         UNWIND $GroupData as group
-        MERGE (g:GSuiteGroup{id: group.id})
+        MERGE (g:GSuiteGroup:GCPPrincipal{id: group.id})
```

Comment:
I think this currently produces a weird neo4j side effect.

It's safer to instead move the additional label to the `set` clause like this: 

```
MERGE (g:GsuiteGroup{id: group.id})
...
SET 
   g.admin_created = <blahblah>,
   ...
   g:GCPPrincipal
```

Example (this is how the query builder generates queries that create multiple labels on a single node): https://github.com/cartography-cncf/cartography/blob/5a296ef5f31458b0010f964e7e42238eb20a29cd/tests/unit/cartography/graph/test_querybuilder_complex.py#L12-L18

Reply:
Will change. What's the side effect?

Reply:
Currently if a node with label GSuiteGroup but without label GCPPrincipal exists, MERGE will attempt to create a new node rather than updating the existing one

Reply:
Ah I see